instrumentation/aws/xray/AwsXrayIdGenerator.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types=1);
+
+namespace Instrumentation\Aws\Xray;
+
+use OpenTelemetry\Sdk\Trace\RandomIdGenerator;
+use OpenTelemetry\Sdk\Trace\IdGenerator;
+
+/**
+ * An ID generator that generates trace IDs that conforms to AWS X-Ray format
+ * Refer to the AWS X-Ray documentation: 
+ * https://docs.aws.amazon.com/xray/latest/devguide/xray-api-sendingdata.html#xray-api-traceids
+ */
+class AwsXrayIdGenerator implements IdGenerator
+{
+    private const TRACE_ID_RANDOM_HEX_LENGTH = 24;
+
+    /**
+     * Constructor creates randomIdGenerator instance
+     */
+    public function __construct() {
+        $this->randomIdGenerator = new RandomIdGenerator();
+    }
+
+    /**
+     * Returns a trace ID in AWS X-Ray format. 
+     * Returns a version number, an epoch timestamp converted to 
+     * 8 hexadecimal digits, and random 96-bit string converted to a
+     * 24 character hexadecimal string, all seperated by 2 dashes.
+     * 
+     * Example: 1-60cd399d-76b435b5411c66f29b238014
```

Comment:
This still needs to be updated to reflect the current implementation.

Reply:
Oops, will update rn!